### PR TITLE
Bug fixes on boostImage() and the Boost event.

### PR DESCRIPTION
**Pull request by Iliannnn**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -41,6 +41,10 @@ export function sleep(milliseconds: number): Promise<void>;
 
 export function welcomeImage(background: string, avatar: string, text_1?: string, text_2?: string, text_3?: string, color?: string, settings?: WelcomeImageSettingOptions): Promise<MessageAttachment>;
 
+export function boostImage(url: string): Promise<string>;
+
+export function start(client: object, boostRoleID: Snowflake): Promise<void>;
```
Comment (reinacchi) [inline index.d.ts]: This as well.

File: functions/boost.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const Discord = require("discord.js");
+
+/**
+ * Starts a boost event
+ * @param {Object} client Your client
```
Comment (reinacchi) [inline functions/boost.js]: Why would a client be an object when it is supposed to be `Discord.Client`?

**Review by reinacchi — CHANGES_REQUESTED** [review]
Typing issues found.

File: functions/boost.js (hunk 2 above)
Comment (Iliannnn) [reply functions/boost.js]: hm lemme see

**Pull request merged**